lib/hubstats/github_api.rb

```
@@ -63,7 +63,7 @@ def self.get_repos
       repos
```

Since org is required, you should throw an exception here instead of returning `repos` which is nil/undefined.